online_meetings/report.py

```diff
@@ -19,7 +19,8 @@
 
 # read configurations
 try:
-    with open(os.path.join(os.path.dirname(__file__), '../config/env.json')) as env_file:        ENV = yaml.safe_load(env_file.read())
+    with open(os.path.join(os.path.dirname(__file__), '../config/env.json')) as env_file:
```

Comment:
Don't know if you want to do this as part of this PR, but it would make sense to use `config/secrets/env.json` instead, since that's where `inventory` expects it to be. I missed that at first and couldn't figure out for a while none of the authentication was working.

Reply:
Oh I didn't notice that, I think this will be refactored completely anyway when the job runner gets committed but I can make this change and move it to secrets.